Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -3616,11 +3623,61 @@ private LatLng getNewLatLng(float fraction, LatLng a, LatLng b) {
         return new LatLng(lat, lng);
     }
 
+
+    private  void animatePolyline(final int staticColor, final int animateColor){
```

Comment:
make most of the things like color, spped, delay .. configurable

Reply:
done